aws-redis-node/variables.tf

```diff
@@ -35,6 +35,12 @@ variable "ingress_security_group_ids" {
   description = "Source security groups which should be able to contact this instance."
 }
 
+variable "ingress_security_group_cidr_blocks" {
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>variable "ingress_security_group_cidr_blocks" is declared but not used